.infra/clickhouse-sync.yml

```diff
@@ -25,7 +25,7 @@ schema.include.list: "public"
 plugin.name: "pgoutput"
 
 # table.include.list: An optional list of regular expressions that match fully-qualified table identifiers for tables to be monitored;
-table.include.list: "public.post,public.source"
+table.include.list: "public.post,public.source,public.user,public.keyword"
```

Comment:
we can't sync user table without filtering some columns. I rather not send personal info to another provider